core/src/main/java/com/netflix/conductor/core/utils/FileInstrumentLogger.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2022 Netflix, Inc.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package com.netflix.conductor.core.utils;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.util.Map;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+public class FileInstrumentLogger extends InstrumentLogger {
+
+    private static final Logger logger = LoggerFactory.getLogger(FileInstrumentLogger.class);
+    private Integer syncObject = Integer.valueOf(1);
+    private BufferedWriter writer = null;
+    private ObjectMapper mapper = new ObjectMapper();
+
+    public void init() {
+        try {
+            writer = new BufferedWriter(new FileWriter("/tmp/performance.log"));
```

Comment:
Please do not hardcode file path and name.

Reply:
sorry i used this as a temp log file. Now I have changed this to /var/log/<podname>/performance.log so that this will also be archived like app.log